api/src/test/java/org/openmrs/OrderTest.java

```diff
@@ -632,6 +662,112 @@ public void isExpired_shouldReturnTrueIfDateStoppedIsNullAndAutoExpireDateIsBefo
 	}
 	
 	/**
+	 * @see Order#isActivated()
+	 * @verifies return true if an order was activated on the current date
+	 */
+	@Test
+	public void isActivated_shouldReturnTrueIfAnOrderWasActivatedOnTheCurrentDate() throws Exception {
+		Order order = new Order();
+		order.setDateActivated(new Date());
```

Comment:
This test doesn't seem correct to me, to guarantee that the dates are the same, you have to create one instance of date and reuse it as dateActivated and checkDate. Otherwise the only reason this test is passing is because by the time isActivated() is called, a new instance of Date is created a couple of milliseconds later which puts checkDate always after dateActivated, feel free to print the dates when comparing them and you will realize this.